Code/Tools/AssetProcessor/native/tests/UnitTestUtilities.h

```diff
@@ -12,6 +12,7 @@
 #include <utilities/assetUtils.h>
 #include <AzCore/Component/ComponentApplicationBus.h>
 #include <AzCore/Interface/Interface.h>
+#include <AzCore/Math/Uuid.h>
```

Comment:
Nit: Any specific reason to include this header? Seems like the new mock structure doesn't need UUID.